src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,49 @@
+import BookmarkModel from '../helpers/bookmarkModel';
+/**
+ * @description class representing Bookmark Controller
+ * @class BookmarkController
+ */
+class BookmarkController {
+  /**
+   * @description - This method is responsible for bookmarking an article
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
```

Comment:
This is false. This method doesn't return anything.

Reply:
OK, I have made necessary corrections. Thank you!